### DISCUSSION TITLE
Adding ADetailer (AD 2nd) to XYZ Plot?

### DISCUSSION THREAD

**Discussion by A-Null-Fissure** [Bing-su/adetailer#674] [Q&A]
Hello, is it possible to add the 2nd+ tabs of ADetailer as options for the XYZ plot?

![2024-07-31_14-56-59](https://github.com/user-attachments/assets/7ac69ca7-1fab-41cb-ace4-6e6fd475c476)

Thanks!

**Comment by Akossimon** [comment]
Yes , i was wondering about these additional tabs as well, they would be extremely useful to be options inside that xyz plot extension

**Comment by coderCK2** [comment]
I was just searching for the same thing! Has it always been like this?

**Comment by gay9cook** [comment]
Please add (AD 2nd, 3rd, 4th, 5th, 6th...etc) to adetailer! ADetailer is only reason why I use Stable Diffusion. thank you so much!
